CIFAR10/inference/BAIDU-USA-GAIT_BaiduNet8_1GV100_pytorch.json

```
@@ -0,0 +1,19 @@
+{
+	    "version": "v1.0",
+	    "author": "Baidu USA GAIT LEOPARD team: Baopu Li, Zhiyu Cheng, Jiazhuo Wang, Haofeng Kou, Yingze Bao", 
+	    "authorEmail": "baopuli@baidu.com, zhiyucheng@baidu.com, jiazhuowang@baidu.com, haofengkou@baidu.com, yingzebao@baidu.com",
+	    "framework": "PyTorch v1.0.1",
+	    "codeURL": " https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8",
+	    "commitHash": "https://github.com/BAIDU-USA-GAIT-LEOPARD/CIFAR10-Inference-BaiduNet8/commit/8678e0bcc72e6b72d5eb66e997e1ae92f2eaf2a1",
+	    "model": "BaiduNet8 using PyTorch JIT in C++",
+	    "hardware": "Baidu Cloud Tesla V100*1/60 GB/12 CPU",
+	    "latency": 0.683,
+	    "cost": 2.078e-3,

```

This doesn't seem correct based on that price. I think you accidentally multiplied by 10,000. In my staging area, this currently shows up as $20.78 USD, which is much higher than any other result we have received.

```suggestion
	    "cost": 2.078e-7,
```

---

Yes, previously we multiplied 10,000 for the whole data. We corrected it as suggested.